packages/web/src/layout/ScreenSize.tsx

```diff
@@ -73,3 +77,27 @@ export function withScreenSize<T>(Component: React.ComponentType<T>) {
     )
   }
 }
+
+// by guessing device type we can have our server rendered content likely be the right size
+// if not it will be fixed on the client after first rendeer.
+// note this is intended to be used serverside and only as a guess
+export function guessDeviceType(userAgent: string) {
```

Comment:
`setDimensionsForScreen`? i would expect a function with the current name to return a device type

Reply:
fair